posthog/api/decide.py

```diff
@@ -185,6 +185,7 @@ def get_decide(request: HttpRequest):
 
             response["capturePerformance"] = True if team.capture_performance_opt_in else False
             response["autocapture_opt_out"] = True if team.autocapture_opt_out else False
+            response["autocaptureExceptions"] = True if team.autocapture_exceptions_opt_in else False
```

Comment:
almost! See below on Line 210 - just add to that test, so no one in the future can bork this - this looks fine :)